company-profiles/karat.md

```diff
@@ -0,0 +1,29 @@
+# Karat
```

Comment:
The company name and filename should match exactly (case sensitive). If you change to karate it should pass the checks :)

Reply:
That isn't why it failed @scove.

Reply:
> That isn't why it failed @scove.

Apologies